File: BOJ/1712.cpp
#include<stdio.h>
int main()
{
	int numb;
	double A, B, C, num;
	scanf("%lf %lf %lf", &A, &B, &C);
	if (C <= B) printf("-1");
	else
	{
		num = A / (C - B);
		numb = (int)num;
		printf("%d\n", numb + 1);
	}
}

File: BOJ(Java 공부용)/1712.java
import java.io.*;
public class Main {
    static BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String[] input = br.readLine().split(" ");
        double a = Double.parseDouble(input[0]);
        double b = Double.parseDouble(input[1]);
        double c = Double.parseDouble(input[2]);
        if(c<=b) System.out.println(-1);
        else {
            double num = a/(c-b);
            int numb = (int)num;
            System.out.println(numb+1);
        }
    }
}